[admin]/admin2/client/main/admin_players.lua

```diff
@@ -479,6 +459,11 @@ function aPlayersTab.onPlayerListScroll(key, state, inc)
     else
         guiGridListSetSelectedItem(aPlayersTab.PlayerList, next, 1)
     end
+
+    -- if we finnally have selected any item
```

Comment:
```suggestion
    -- If we finally have selected an item
```

Is this `if` check needed? It seems that by the time we get to this line, we will always have selected an item.